### ISSUE TITLE
Regarding benchmark results

### ISSUE THREAD

**Issue by Alighieri1231** [MacDunno/TinyUSFM#1]
<img width="928" height="383" alt="Image" src="https://github.com/user-attachments/assets/74e6263b-da98-489e-a78b-871d42965a81" />

<img width="516" height="84" alt="Image" src="https://github.com/user-attachments/assets/1f0e9449-0df0-4bc7-b8bf-1dd1e61f7d08" />

Hi, could you explain what BUSI seed did you use for the experiments and what VIT (I am using the timm deit 1k) and/or resnet50(torch models) are you using? I am having very low results compared to your benchmark. 

Thanks!

**Comment by MacDunno** [comment]
@Alighieri1231 
Hi, thanks for your interest in our work!

Here are some additional details regarding the benchmark settings:

* **ViT-T** was trained **from scratch with random initialization**, without any pretrained weights.
* **ResNet** was initialized using the **ImageNet-1K pretrained weights provided by the ****`timm`**** library**.
* The **random seed was set to 1234** for our experiments.
* For **BUSI classification**, the original BUSI release does not provide an official train/validation/test split. Therefore, following the UniUS-Bench protocol, we used a **stratified random split of 7:1:2 (train/validation/test)**, with the random seed set to 1234.

Hope this clarifies the benchmark settings. Thanks again for your interest in TinyUSFM!
